src/components/App.tsx

```diff
@@ -47,10 +48,13 @@ class App extends TsxComponent<AppProps> {
   @Prop({ required: true }) defaultLocale!: AppProps["defaultLocale"];
   @Prop({ required: true }) handleRouteChange!: AppProps["handleRouteChange"];
   @Prop() fsTppVersion: AppProps["fsTppVersion"];
+  @Prop({ default: false }) customSnapHooks!: AppProps["customSnapHooks"];
```

Comment:
Additionally, I think we need to add a short documentation regarding these default hooks and how to override them here:  https://github.com/e-Spirit/fsxa-pwa/blob/master/docs/modules/ROOT/pages/advanced/TPP-Snap.adoc

Reply:
Please doublecheck https://github.com/e-Spirit/fsxa-pwa/pull/121.